app/src/main/java/com/dreamisi/moviebase/movieList/MoviesListViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.dreamisi.moviebase.movieList
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.dreamisi.moviebase.data.MovieRepository
+import com.dreamisi.moviebase.data.models.Movie
+import kotlinx.coroutines.launch
+
+class MoviesListViewModel(
+
+    private val myRepository: MovieRepository,
+
+    ) : ViewModel() {
```

Comment:
Поля в конструкторе обычно указываю без дополнительный отступов
```suggestion
class MoviesListViewModel(
    private val myRepository: MovieRepository,
) : ViewModel() {
```